openwisp_controller/subnet_division/base/models.py

```diff
@@ -139,11 +146,10 @@ def _validate_master_subnet_consistency(self):
             )
 
     def _validate_ip_address_consistency(self):
-        # Validate individual generated subnet can accommodate required number of IPs
         try:
             next(
                 ip_network(str(self.master_subnet.subnet)).subnets(new_prefix=self.size)
-            )[self.number_of_ips]
+            )[self.number_of_ips - 1]
```

Comment:
/32 and /128 subnets only have the zero index, trying to access 1 fails, therefore I had to subtract 1 here.